web-registry/src/components/organisms/SellOrdersActionsBar/SellOrdersActionsBar.tsx

```diff
@@ -16,46 +18,71 @@ type Params = {
   isBuyButtonDisabled: boolean;
   onSellButtonClick: () => void;
   onBuyButtonClick: () => void;
+  onChainProjectId?: string;
+  projectName?: string;
+  onChainCreditClassId?: string;
+  creditClassName?: string;
 };
 
 export const SellOrdersActionsBar = ({
   isSellButtonDisabled,
   isBuyButtonDisabled,
   onSellButtonClick,
   onBuyButtonClick,
-}: Params): JSX.Element => (
-  <StickyBar>
-    <Box sx={{ display: 'flex', justifyContent: 'flex-end', width: '100%' }}>
-      <InfoTooltip
-        title={isSellButtonDisabled ? SELL_DISABLED_TOOLTIP : ''}
-        arrow
-        placement="top"
-      >
-        <span>
-          <OutlinedButton
-            onClick={onSellButtonClick}
-            disabled={isSellButtonDisabled}
-            sx={{ mr: 5 }}
-          >
-            {'SELL'}
-          </OutlinedButton>
-        </span>
-      </InfoTooltip>
-      <InfoTooltip
-        title={isBuyButtonDisabled ? BUY_DISABLED_TOOLTIP : ''}
-        arrow
-        placement="top"
-      >
-        <span>
-          <ContainedButton
-            startIcon={<CurrentCreditsIcon height="18px" width="18px" />}
-            onClick={onBuyButtonClick}
-            disabled={isBuyButtonDisabled}
-          >
-            {'BUY CREDITS'}
-          </ContainedButton>
-        </span>
-      </InfoTooltip>
-    </Box>
-  </StickyBar>
-);
+  onChainProjectId,
+  projectName,
+  onChainCreditClassId,
+  creditClassName,
+}: Params): JSX.Element => {
+  const location = useLocation();
+  const { track } = useAnalytics();
+  return (
+    <StickyBar>
+      <Box sx={{ display: 'flex', justifyContent: 'flex-end', width: '100%' }}>
+        <InfoTooltip
+          title={isSellButtonDisabled ? SELL_DISABLED_TOOLTIP : ''}
+          arrow
+          placement="top"
+        >
+          <span>
+            <OutlinedButton
+              onClick={onSellButtonClick}
+              disabled={isSellButtonDisabled}
+              sx={{ mr: 5 }}
+            >
+              {'SELL'}
+            </OutlinedButton>
+          </span>
+        </InfoTooltip>
+        <InfoTooltip
+          title={isBuyButtonDisabled ? BUY_DISABLED_TOOLTIP : ''}
+          arrow
+          placement="top"
+        >
+          <span>
+            <ContainedButton
+              startIcon={<CurrentCreditsIcon height="18px" width="18px" />}
+              onClick={() => {
+                // if any of the values used in event data are undefined,
+                // then they are not included in the tracking data.
+                // this is a feature of the track API.
+                track('buy1', {
```

Comment:
this track call covers the sticky nav buy button (in other words, project page and credit class page buy buttons)